taxonomies/food/ingredients.txt

```diff
@@ -30526,6 +30535,11 @@ sv: riskrisp, puffat ris
 # ingredient/fr:riz-croustillant has 44 products in 2 languages @2018-12-19
 # usage:en:crisped rice (Rice, Sugar, Salt, Barley Malt Extract)
 
+< en:corn crisps
+< en:puffed rice
+en: rice and corn crisps
+sv: ris- och majskrisp
```

Comment:
This one, you can leave it as it is because I don't think the parsing will work with ingredients_processing.

Reply:
yes, good question. It can inherit from **< en: corn** instead of **< en: corn crisps**.